beacon-chain/sync/rpc_goodbye.go

```diff
@@ -2,22 +2,36 @@ package sync
 
 import (
 	"context"
+	"fmt"
 	"time"
 
 	libp2pcore "github.com/libp2p/go-libp2p-core"
-	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
 )
 
+var goodByes = map[uint64]string{
```

Comment:
Can we define a status code enum too?

```go
const (
	codeClientShutdown uint64 = iota
	codeWrongNetwork
	codeGenericError
)
```

Then maybe you can do this

```go
var goodByes = map[uint64]error{
	codeClientShutdown: errors.New("client shutdown"),
	...
}
```
These should be lowercase strings too